tests/test_subset.py

```diff
@@ -1688,6 +1687,98 @@ def test_get_time_OMI(self):
             assert "Time" in time_var_names[0]
             assert "Latitude" in lat_var_names[0]
 
+    def test_sndr_dims(self):
```

Comment:
It seems like this test contains a lot of duplicate code from the subsetter itself -- can you call a function from the subsetter and make assertions on the result rather than doing it this way?

Reply:
Trying to minimize calling the entire subset.subset method and just calling the subset_with_bbox method. Will just add assertions to the dimensions input vs output